src/si/electric_current.rs

```diff
@@ -40,6 +40,8 @@ quantity! {
         @yoctoampere: prefix!(yocto); "yA", "yoctoampere", "yoctoamperes";
 
         @abampere: 1.0_E1; "abA", "abampere", "abamperes";
+        @atomic_unit_of_charge_per_second: 1.602_176_634_E-19; "a.u. of charge/s",
+            "atomic unit of charge per second", "atomic units of charge per second";        
```

Comment:
Should elementary charge per second exist? If so, should probably add a comment.

```suggestion
        /// ...
        @elementary_charge_per_second: 1.602_176_634_E-19; "e/s", "elementary charge per second",
            "elementary charges per second";
        @atomic_unit_of_charge_per_second: 1.602_176_634_E-19; "a.u. of charge/s",
            "atomic unit of charge per second", "atomic units of charge per second";
        @abampere: 1.0_E1; "abA", "abampere", "abamperes";
```